Service accounts: update docs (#52734)
grafana/grafana: docs/sources/administration/service-accounts/_index.md

You can assign on of the following permissions to a specific user or a team:

> 1. **Edit**: User or a team with this permission will be able to view, edit, enable and disable service account, as well as view, add or delete service account tokens.

Reviewer: ```suggestion
1. **Edit**: A user or a team with this permission can view, edit, enable and disable a service account, and add or delete service account tokens.
```
